tests/iele/unit/logarithm.iele.json

```diff
@@ -0,0 +1,123 @@
+                        "logs": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347", 
+                        "refund": ""
+                    },
+                    {
+                        "out": ["0x01"],
+                        "status": "",
+                        "gas": "0x0fa92d",
+                        "logs": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347", 
+                        "refund": ""
+                    },
+                    {
+                        "out": ["0x02"],
+                        "status": "",
+                        "gas": "0x0fa92d",
+                        "logs": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347", 
+                        "refund": ""
+                    }
+                ],
+                "transactions": [
+                    {
+                        "nonce": "0x00", 
+                        "function": "test_logarithm", 
+                        "gasLimit": "0x100000", 
+                        "value": "0x00", 
+                        "to": "0x1000000000000000000000000000000000000000", 
+                        "arguments": ["0x01"], 
+                        "contractCode": "",
+                        "gasPrice": "0x01",
+                        "from": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
+                    },
+                    {
+                        "nonce": "0x00", 
+                        "function": "test_logarithm", 
+                        "gasLimit": "0x100000", 
+                        "value": "0x00", 
+                        "to": "0x1000000000000000000000000000000000000000", 
+                        "arguments": ["0x02"], 
+                        "contractCode": "",
+                        "gasPrice": "0x01",
+                        "from": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
+                    },
+                    {
+                        "nonce": "0x00", 
+                        "function": "test_logarithm", 
+                        "gasLimit": "0x100000", 
+                        "value": "0x00", 
+                        "to": "0x1000000000000000000000000000000000000000", 
+                        "arguments": ["0x03"], 
+                        "contractCode": "",
+                        "gasPrice": "0x01",
+                        "from": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
+                    },
+                    {
+                        "nonce": "0x00", 
+                        "function": "test_logarithm", 
+                        "gasLimit": "0x100000", 
+                        "value": "0x00", 
+                        "to": "0x1000000000000000000000000000000000000000", 
+                        "arguments": ["0x04"], 
+                        "contractCode": "",
+                        "gasPrice": "0x01",
+                        "from": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
```

Comment:
There should be another test on input larger than a byte, maybe something larger than 64 bits.